bin/check-http.rb

```diff
@@ -250,10 +263,19 @@ def run
       config[:port] ||= config[:ssl] ? 443 : 80
     end
 
+    # Use Ruby DNS Resolver and set DNS resolution timeout to 800ms
+    dns_resolver = Resolv::DNS.new
+    dns_resolver.timeouts = 0.8
```

Comment:
ya let's expose that as an option so its configurable and make sure to cast it as a float.